routes/asset.js

```diff
@@ -70,25 +55,69 @@ router.get('/:id', (req, res) => {
   }
 });
 
-// // Route to Update asset name in database
-// router.get('/update/:id', (req, res) => {
-//   let newName = 'Updated_Asset_Name!';
-//   let sql = `UPDATE asset SET name = '${newName}' WHERE id = ${req.params.id}`;
-//   let query = connection.query(sql, (err, result) => {
-//     if (err) throw err;
-//     console.log(result);
-//     res.send(JSON.stringify(result));
-//   });
-// });
+// POST request to Create an asset
+router.post('/', (req, res) => {
+  let mysql = req.app.get('mysql');
+  let sql =
+    'INSERT INTO asset (name, physical_file_name, uploader, physical_file_size, physical_file_type) VALUES (?, ?, ?, ?, ?)';
+  let inserts = [
+    req.body.name,
+    req.body.fileName,
+    req.body.uploader,
+    req.body.fileSize,
+    req.body.fileType
+  ];
+  sql = mysql.pool.query(sql, inserts, (err, result) => {
```

Comment:
You should avoid reassigning variables unless you really need to. And, even then, you shouldn't reassign a value of a different type.
In this case, since you're not using the `sql` variable anywhere below here, you don't actually need to assign the output of `mysql.pool.query` to a variable.
You can just run it like a standalone function and when it's done it calls the callback function you provided.